lib/DxilPIXPasses/DxilAnnotateWithVirtualRegister.cpp

```diff
@@ -107,6 +107,29 @@ void DxilAnnotateWithVirtualRegister::applyOptions(llvm::PassOptions O) {
 
 char DxilAnnotateWithVirtualRegister::ID = 0;
 
+static const char* ShaderKindName(hlsl::DXIL::ShaderKind kind) {
```

Comment:
Other passes may need this, could we move it to DxilShaderModel.cpp?

Reply:
+1 to @python3kgae's suggestion. We have (at least) one other implementation of this logic in the `D3DReflectionDumper`. I think this implementation is cleaner (although clang-format would be nice).

Reply:
I have now discovered that there already is such a function! Deleted mine, and used the existing one.